1-mandatory/4-budgets.js

```diff
@@ -10,7 +10,20 @@ getBudgets([
 Should give return the answer of 62600.
 **/
 
```

Comment:
This is a nice attempt - but I wonder if you can simplify this a bit?
Can you think of a way to do this using only the array `reduce` method? (so no extra for loops)